### ISSUE TITLE
Request: adding livenessProbe parameters to Helm template.

### ISSUE THREAD

**Issue by mk811** [jenkinsci/kubernetes-operator#498]
# Expected Behavior

Jenkins Master pod can be initialized at the stable way.

# Actual Behavior

Since we have bunch of plugins to be installed and have limited CPU / memory resources for Jenkins Master, initialization process takes long time. We faced LivenessProbe killed Jenkins Master pod during the initialization process.

# Steps to Reproduce the Problem

1. Place a lot of plugins randomly under `plugins` parameter, which takes more than 200 seconds  `initialDelaySeconds` 80 + `failureThreshold` 12 * `periodSeconds` 10. 
2. Start up Jenkins Master pod
3. After 200 seconds, Jenkins Master pod will be restered.

# Additional Info

- Jenkins Operator version:  0.3

# Request

I would like to make `livenessProbe` and `readinessProbe` configurable parameter in Helm template as below. The default will be as-is configuration. if you like it, I'm happy to commit and raise a PR. 

```yaml
# kubernetes-operator/chart/jenkins-operator/templates/jenkins.yaml
{{- with .Values.jenkins.livenessProve }}
livenessProve: {{ toYaml . | nindent 10 }}
{{- end }}
{{- with .Values.jenkins.readinessProbe }}
readinessProbe: {{ toYaml . | nindent 10 }}
{{- end }}
```

```yaml
# kubernetes-operator/chart/jenkins-operator/values.yaml
jenkins:
  livenessProbe:
    failureThreshold: 12
    httpGet:
      path: /login
      port: http
      scheme: HTTP
    initialDelaySeconds: 80
    periodSeconds: 10
    successThreshold: 1
    timeoutSeconds: 5
  readinessProbe:
    failureThreshold: 3
    httpGet:
      path: /login
      port: http
      scheme: HTTP
    initialDelaySeconds: 30
    periodSeconds: 10
    successThreshold: 1
    timeoutSeconds: 1
```

**Comment by tomaszsek** [comment]
Hi @mk811,

Your PR is more than welcome :+1: . We will take care of releasing a new version of Helm chart.

Cheers

**Comment by tomaszsek** [comment]
Released in 0.4.1 Helm chart
